static/js/dom.js

```diff
@@ -122,5 +124,18 @@ export let dom = {
             }
             dom.getBoardIdsFromDocument();
         }
+    },
+
+    renameBoard: function () {
+        let boardTitle = document.querySelectorAll('.board-title');
+        for (let title of boardTitle) {
+            title.addEventListener('dblclick', function (event) {
+                let original = event.currentTarget;
+                event.currentTarget.outerHTML = `<input type="text" class="new-board-title" value="${original.textContent}" size="15" required minlength="1" >` +
+                    `<button class="title-save-button">Save</button>`;
```

Comment:
When using the template literals, you can write multiline strings. So you don't need to close the string, write a `+` sign, and start a new string, just hit enter in the string anywhere and continue writing it in the next line.